scripts/procedures/getConfig.ts

```diff
@@ -594,48 +598,45 @@ export const [getConfig, setConfigMatcher] = compat.getConfigAndMatcher({
         type: "object",
         name: "Plugins",
         description:
-          "Plugins are subprocesses that provide extra functionality and run alongside the lightningd process inside \nthe main Core Lightning container in order to communicate directly with it.\nTheir source is maintained separately from that of Core Lightning itself.\n",
+          "Plugins are subprocesses that provide extra functionality and run alongside the lightningd process inside \nthe main Core Lightning container in order to communicate directly with it.\nTheir source is maintained separately from that of Core Lightning itself.",
         spec: {
           rebalance: {
             type: "boolean",
             name: "Rebalance",
             description:
-              "Enables the `rebalance` rpc command, which moves liquidity between your channels using circular payments.\nSee `help rebalance` on the CLI or in the Spark console for usage instructions.\n\nSource: https://github.com/lightningd/plugins/tree/master/rebalance\n",
+              "Enables rebalancing, which moves liquidity between your channels using circular payments.  As of early 2024, this can only be done in the CLI.  <b>Default: False</b><br><b>Source:  https://github.com/lightningd/plugins/tree/master/rebalance</b>",
```

Comment:
To my reading, this plugin was always only usable via CLI, unless there was an client that was able to make this RPC call that is no longer supported/available.

https://github.com/lightningd/plugins/tree/master/rebalance

Reply:
Spark was a "UI," where you could enter these calls, which is now deprecated.  I'm simply making a note that this is not possible in any UI tools at this time.  Hopefully RTL will pick up the ball here